includes/class-wc-calypso-bridge-notes.php

```diff
@@ -41,16 +41,24 @@ public function __construct() {
 	 * Include notes and initialize note hooks.
 	 */
 	public function init() {
-		include_once dirname( __FILE__ ) . '/notes/class-wc-calypso-bridge-navigation-learn-more-note.php';
+		if ( wc_calypso_bridge_is_ecommerce_plan() ) {
+			include_once dirname( __FILE__ ) . '/notes/class-wc-calypso-bridge-navigation-learn-more-note.php';
+			new WC_Calypso_Bridge_Navigation_Learn_More_Note();
+		}
 
-		$navigation_learn_more_note = new WC_Calypso_Bridge_Navigation_Learn_More_Note();
+		include_once dirname( __FILE__ ) . '/notes/class-wc-calypso-bridge-payments-remind-me-later-note.php';
```

Comment:
I agree with the current way of importing scripts isn't ideal. Though I also noticed that the current composer setup does not actually include any dependencies that are used by bridge, it's merely tooling for running PHPUnit and other stuff used for development. In fact, `./vendor/autoload.php` isn't even included in bridge's current runtime.

I feel that this is much better to be tackled outside of this PR since the change can be reviewed with better focus (maybe it should be a great cooldown issue?), and let the unusual way of importing be a poor man's JIT if that's okay with you. 😃

Reply:
Sounds good! Yeah let's just take the unusual way in this PR.

I'll create an issue for it. I agree it would be a great thing to tackle during a cooldown.